src/web/app/pages-static/about-page/about-page.component.scss

```diff
@@ -16,6 +16,19 @@
 }
 
 .core-team-details-cell {
-  min-width: 360px;
   padding-right: 40px;
 }
+
+ul {
+  list-style-position: inside;
+}
+
+ol {
+  list-style-position: inside;
+}
+
+table {
+  margin-left: 10px;
+  margin-right: 10px;
```

Comment:
Same here, margins can be removed if we add padding to the main content for all pages.